gtsfm/scene_optimizer.py

```diff
@@ -346,6 +358,9 @@ def aggregate_frontend_metrics(
     success_count_unit3 = np.sum(unit3_errors < angular_err_threshold_deg)
     success_count_pose = np.sum(pose_errors < angular_err_threshold_deg)
 
+    # count entries with all correct correspondences
```

Comment:
i might just add to the inline comment that we are checking which rows have inlier ratio == 1

Reply:
done.